src/capturer/engine/mac/mod.rs

```diff
@@ -90,22 +88,23 @@ pub fn create_capturer(options: &Options, tx: mpsc::Sender<Frame>) -> SCStream {
         .windows
         .into_iter()
         .filter(|window| {
-            if let Some(excluded_window_names) = &options.excluded_windows {
```

Comment:
I have a snippet fix for this ready. Will apply it to Vasu's PR since this part of the code depends on how he structures it.